docs/utilities/parameters.md

```diff
@@ -516,6 +516,22 @@ The **`config`** and **`boto3_session`** parameters enable you to pass in a cust
 		...
 	```
 
+=== "Custom client"
+
+	```python hl_lines="2 4 5"
+	from aws_lambda_powertools.utilities import parameters
+	import boto3
+
+	boto3_client= boto3.client("ssm")
+	ssm_provider = parameters.SSMProvider(boto3_client=boto3_client)
+
+	def handler(event, context):
+		# Retrieve a single parameter
+		value = ssm_provider.get("/my/parameter")
+		...
```

Comment:
I would update the example to still use boto session to create the client  (see [boto3 Sessions, and Why You Should Use Them](https://ben11kehoe.medium.com/boto3-sessions-and-why-you-should-use-them-9b094eb5ca8e) ). And showcase what you can do when you create your own client, like customerize the endpoint url as per the original issue.

```suggestion
	from aws_lambda_powertools.utilities import parameters
	import boto3

        session = boto3.session.Session()
	boto3_client= session.client(service_name="ssm", endpoint_url='custom_endpoint')
	ssm_provider = parameters.SSMProvider(boto3_client=boto3_client)

	def handler(event, context):
		# Retrieve a single parameter
		value = ssm_provider.get("/my/parameter")
		...
```